dyno-queues-redis/src/main/java/com/netflix/dyno/queues/redis/RedisDynoQueue.java

```diff
@@ -182,24 +204,112 @@ public int getUnackTime() {
             if (ids == null) {
                 return Collections.emptyList();
             }
+            return doPeekBodyHelper(ids);
 
-            List<Message> msgs = execute("peek", messageStoreKey, () -> {
-                List<Message> messages = new LinkedList<Message>();
-                for (String id : ids) {
-                    String json = nonQuorumConn.hget(messageStoreKey, id);
-                    Message message = om.readValue(json, Message.class);
-                    messages.add(message);
-                }
-                return messages;
-            });
+        } finally {
+            sw.stop();
+        }
+    }
+
+    @Override
+    public List<Message> unsafe_peekAllShards(final int messageCount) {
```

Comment:
Can we follow camelCase since this is java.

Reply:
Done.